refarch-integrations/refarch-address-integration/refarch-address-integration-core/src/test/java/de/muenchen/refarch/integration/address/application/usecase/StreetsMunichUseCaseTest.java

```diff
@@ -0,0 +1,75 @@
+package de.muenchen.refarch.integration.address.application.usecase;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import de.muenchen.refarch.integration.address.application.port.in.StreetsMunichInPort;
+import de.muenchen.refarch.integration.address.application.port.out.AddressClientOutPort;
+import de.muenchen.refarch.integration.address.client.exception.AddressServiceIntegrationException;
+import de.muenchen.refarch.integration.address.client.model.request.ListStreetsModel;
+import de.muenchen.refarch.integration.address.client.gen.model.Strasse;
+import de.muenchen.refarch.integration.address.client.gen.model.StrasseResponse;
+import org.junit.jupiter.api.Test;
+
+class StreetsMunichUseCaseTest {
+
+    public static final String SOME_ERROR = "SomeError";
+    public static final String MESSAGE_400 = "400";
+    private final AddressClientOutPort addressClientOutPort = mock(AddressClientOutPort.class);
+
+    private final StreetsMunichInPort streetsMunichUseCase = new StreetsMunichUseCase(addressClientOutPort);
+
+    @Test
+    void testFindStreetsByIdReturnsStrasse() throws AddressServiceIntegrationException {
```

Comment:
Why are the Test methods annotated with `throws`? Afaik this is bad practice. Testing for exception should always be done via assertions like shown in https://stackoverflow.com/a/2935935. This applies to many more test methods in this PR.

Reply:
Why should it?
In my opinion that's ok as I don't test that a exception should be thrown but when the exception is thrown the test fails.